modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
Hi @natocesarrego, I don't know why this commit is in this order. I rebased and put it to be the last commit =/ Both in source tree and git log it's in the correct order. Sorry for that.

Reply:
No problem, @alinedoleron. 🙂 It already happened to me too. Maybe it occurs due to the differences among the date and time of the commits. Anyway, thanks for your advance clarification. 😉